lib/events/pgevents/pgevents.go

```diff
@@ -595,3 +589,37 @@ func (l *Log) GetEventExportChunks(ctx context.Context, req *auditlogpb.GetEvent
 func (l *Log) SearchSessionEvents(ctx context.Context, req events.SearchSessionEventsRequest) ([]apievents.AuditEvent, string, error) {
 	return l.searchEvents(ctx, req.From, req.To, events.SessionRecordingEvents, req.Cond, req.SessionID, req.Limit, req.Order, req.StartKey)
 }
+
+// sessionIDBase is a randomly-generated UUID used as the basis for deriving
+// an UUID from session IDs. See Log.parseSessionID.
```

Comment:
```suggestion
// sessionIDBase is a randomly-generated UUID used as the basis for deriving
// an UUID from session IDs. See [Log.parseSessionID].
```

Reply:
The link won't link in here, this is a private var.